Runtime/Core/SelectionPickerSettings.cs

```diff
@@ -1,70 +1,101 @@
 using System;
+using System.Collections.Generic;
 
 namespace UnityEngine.ProBuilder
 {
     public class SceneSelection : IEquatable<SceneSelection>
     {
         public GameObject gameObject;
         public ProBuilderMesh mesh;
-        public int vertex;
-        public Edge edge;
-        public Face face;
+        public List<int> vertexes;
+        public List<Edge> edges;
+        public List<Face> faces;
+
```

Comment:
I'm not so sure about this part. I did this to simplify cases when we want to set the list to a single value, but it may be confusing?

Reply:
indeed, maybe a clear function name instead

Reply:
Agreed, either change these to a `Set{Vertexes, Edges, Faces}` function or remove.